src/alias/azext_alias/custom.py

```diff
@@ -0,0 +1,202 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import re
+import shlex
+import hashlib
+
+from knack.util import CLIError
+
+from azext_alias import AliasCache
+from azext_alias.argument import get_placeholders
+from azext_alias._const import (
+    COLLISION_CHECK_LEVEL_DEPTH,
+    ALIAS_NOT_FOUND_ERROR,
+    INVALID_ALIAS_COMMAND_ERROR,
+    EMPTY_ALIAS_ERROR,
+    INVALID_STARTING_CHAR_ERROR,
+    INCONSISTENT_ARG_ERROR,
+    COMMAND_LVL_ERROR
+)
+from azext_alias.alias import (
+    GLOBAL_ALIAS_PATH,
+    get_config_parser,
+    AliasManager
+)
+
+
+def create_alias(alias_name, alias_command):
+    """
+    Create an alias.
+
+    Args:
+        alias_name: The name of the alias.
+        alias_command: The command that the alias points to.
+    """
+    alias_name, alias_command = alias_name.strip(), alias_command.strip()
+    _validate_alias_name(alias_name)
+    _validate_alias_command(alias_command)
```

Comment:
This validation should happen in a _validators.py file.
See others in the CLI for examples.